src/Symfony/Component/Security/Core/Validator/Constraints/UserPasswordValidator.php

```diff
@@ -39,6 +39,10 @@ public function validate($password, Constraint $constraint)
             throw new UnexpectedTypeException($constraint, __NAMESPACE__.'\UserPassword');
         }
 
+        if (null === $password || '' === $password) {
+            return;
```

Comment:
Probably because you ran it on PHP lower than 5.6, before function [hash_equals](http://php.net/manual/en/function.hash-equals.php) was used. If that gets a null in comparison, it breaks.